source/eitri/eitri.py

```diff
@@ -44,43 +52,121 @@
+    :return: the output file path
+    :return: the path of the folder containing all "*2ed" and "ed2nav" binaries
+    :return: the path to the binary "cities"
+    :return: the path to cities file (ex: france_boundaries.osm.pbf) to load
     """
+    parser = argparse.ArgumentParser(
+        prog='eitri',
+        description='Eitri uses a set of binaries to extract data, load it into a database and extract it to a single file. '
+        'This software is not distributed and only used by Navitia.',
+        epilog='For more informations, see www.navitia.io',
+    )
+    parser.add_argument(
+        '-d',
+        '--data-folder',
+        required=True,
+        type=is_folder,
+        help='the directory path with all data. If several datasets ("*.osm", "*.gtfs", ...) are available, they need to be in separate directories',
+    )
+    parser.add_argument(
+        '-e',
+        '--ed-folder',
+        required=True,
+        type=is_folder,
+        help='the directory path containing all "*2ed" and "ed2nav" binaries',
+    )
+    parser.add_argument(
+        '-o',
+        '--output-file',
+        type=is_file_exist,
+        default='./data.nav.lz4',
+        help="the output file path (default: './data.nav.lz4')",
+    )
+    parser.add_argument(
+        '-i', '--cities-file', type=is_file, help='the path to the "cities" file to load  (usually a *.osm.pbf)'
+    )
+    parser.add_argument(
+        '-f', '--cities_folder', type=is_folder, help='the path to the folder containing the "cities" binary'
+    )
+    parser.add_argument(
+        '-c', '--color', type=is_enabled, default=True, help='enable or disable colored logs (default: True)'
+    )
+    return parser.parse_args()
 
-    # there is some problems with environment variables and cmake, so all args can be given through cli
-    for p in add_pythonpath:
-        sys.path.append(p)
 
-    from ed_handler import generate_nav
-    from docker_wrapper import PostgresDocker
+def eitri():
+    # type: () -> None
+    """
+    Generate a "data.nav.lz4" file.
+    """
+    args = get_params()
+    if args.color:
+        log.enable_color(True)
+    else:
+        log.enable_color(False)
 
-    with closing(PostgresDocker()) as docker_ed, closing(PostgresDocker()) as docker_cities:
-        generate_nav(
-            data_dir,
+    logger.info("starting Eitri")
+    with contextlib.closing(docker_wrapper.PostgresDocker()) as docker_ed, contextlib.closing(
+        docker_wrapper.PostgresDocker()
+    ) as docker_cities:
+        ed_handler.generate_nav(
+            args.data_folder,
             docker_ed,
             docker_cities,
-            output_file,
-            ed_component_path=ed_component_path,
-            cities_exec_path=cities_exec_path,
-            import_cities=import_cities,
+            args.output_file,
+            args.ed_folder,
+            args.cities_file,
+            args.cities_folder,
         )
+    logger.info("everything is done !")
```

Comment:
```suggestion
    logger.info("data.nav file generated !")
```

Reply:
"data.nav" is not a constant and this variable is displayed the next line : https://github.com/CanalTP/navitia/pull/2852/files/5a86c50102bcdd7e16e42e697f49d72f4d078bbe?file-filters%5B%5D=.py&hide-deleted-files=true#diff-7b35b6684c81d9c6225c1fda49e7d26bR168

Reply:
Fine, let's find something else, but more useful than 'everything's done'

Reply:
It just a general information to reassure the user, the nthe second is a reminder of the result path.

Do you prefere to remove the first ?

Reply:
Removed